tests/with_client_cert.yaml

```diff
@@ -75,6 +75,7 @@
         - name: Generate client/server certificates
           block:
             - name: Set the CSR file
+              set_fact:
```

Comment:
I added what was likely meant to be a set_fact task, although I'm not sure if ``with_items`` works inside a block ?
That would be news to me, I used to work around that limitation by including a tasks file instead.

Reply:
Oh yeah, I don't think it does. I think I've had to use `include_tasks` like you said.

Reply:
Yup:
```
2022-08-03 19:42:19.307603 | ANSIBLE PARSE ERROR
2022-08-03 19:42:19.307636 | Using /var/lib/zuul/builds/647fe92e2c064741af70f2a0d4c48774/ansible/playbook_0/ansible.cfg as config file
2022-08-03 19:42:19.307652 | [WARNING]: Invalid characters were found in group names but not replaced, use
2022-08-03 19:42:19.307662 | -vvvv to see details
2022-08-03 19:42:19.307672 | ERROR! 'with_items' is not a valid attribute for a Block
2022-08-03 19:42:19.307684 | 
2022-08-03 19:42:19.307693 | The error appears to be in '/var/lib/zuul/builds/647fe92e2c064741af70f2a0d4c48774/untrusted/project_0/github.com/ansible-community/ara-collection/tests/with_client_cert.yaml': line 75, column 11, but may
2022-08-03 19:42:19.307705 | be elsewhere in the file depending on the exact syntax problem.
```

I'll figure something out.

Reply:
Since it's really just two items, I split the block in two. It's not very DRY but it's not a big deal.